set_delivery_address/models/sale_order.py

```diff
@@ -0,0 +1,31 @@
+from odoo import models, api
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
+        super(SaleOrder, self).onchange_partner_id()
```

Comment:
No need to call super. @api.onchange decorator itself replaces the onchange behavior for the specified field. Also, add doc string comment